language-extensions/dotnet-core-CSharp/sample/LoopBackConnection/LoopBackConnectionOLEDB.cs

```diff
@@ -0,0 +1,69 @@
+//*********************************************************************
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+//
+// @File: LoopBackConnectionOLEDB.cs
+//
+// Purpose:
+//  Connect to SQL server using the loopback connection in a Language extension environment.
+//
+//*********************************************************************
+using Microsoft.Data.Analysis;
+using Microsoft.Data.SqlClient;
+using Microsoft.SqlServer.CSharpExtension.SDK;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+using System.Data.OleDb;
+
+namespace UserExecutor
+{
+    /// <summary>
+    /// This class extends the AbstractSqlServerExtensionExecutor and uses
+    /// a regular expression that checks if a text contains the word "C#" or "c#"
+    /// </summary>
+    public class LoopBackConnectionOLEDB : AbstractSqlServerExtensionExecutor
+    {
+        /// <summary>
+        /// This method overrides the Execute method from AbstractSqlServerExtensionExecutor.
+        /// </summary>
+       
+        public override DataFrame Execute(DataFrame input, Dictionary<string, dynamic> sqlParams){
+
+            //Ole DB implementation
+            // Connection string to the server.. 
+            // This is a standard connection string. 
+            // Example: 
+            // string connectionstring = "Data Source=SQLSERVER2022;User Id=user1;Password=xxxx;Initial Catalog=DBNAME;Trusted_Connection=True;Encrypt=False;";
+            string connectionstring = "Data Source=<ServerName>;User Id=<<UserID>>;Password=<<Credentials>>;Initial Catalog=<<Database>>;Trusted_Connection=True;Encrypt=False;";
+
+            using (OleDbConnection connection = new OleDbConnection(connectionstring))
+            {
+
+                connection.Open();
+                // The SQL command that you need to execute on the SQL server.
+                // This can be passed as a parameter as well into this method if you want to make it more dynamic.
+                String sql = "SELECT field1, field2 FROM table1";
+
+
+                using (OleDbCommand command = new OleDbCommand(sql, connection))
+                {
+                    using (OleDbDataReader reader = command.ExecuteReader())
+                    {
+                        while (reader.Read())
+                        {
+                            Console.WriteLine("{0} {1}", reader.GetString(0), reader.GetString(1));
+                        }
+                    }
+                }
+                
+                }
```

Comment:
nit: indentation

Reply:
Have fixed the Indentation in the latest commit.